packages/lumx-react/src/components/thumbnail/IFocus.ts

```diff
@@ -0,0 +1,5 @@
+// Credits: https://github.com/third774/image-focus/
+export interface IFocus {
```

Comment:
Could use a more generic name like `Point2D` (Some similar interface are probably used in `useComputePosition`)

Reply:
I think we must have a specific interface, indeed they are coordinates but limited from -1 to 1 so if we use specific interface we know there are limitations